docs/api/react-native-windows.datepicker._constructor_.md

```diff
@@ -16,5 +16,5 @@ constructor(props: IDatePickerProps);
 
 |  Parameter | Type | Description |
 |  --- | --- | --- |
-|  props | <code>IDatePickerProps</code> |  |
+|  props | [IDatePickerProps](./react-native-windows.idatepickerprops.md) |  |
```

Comment:
What's with all the md file changes?

Reply:
This is a consequence of people not running `yarn api` more often. These should just be formatting changes, but they've been lingering around for a while and no one has gotten around to checking them in.

Reply:
@NickGerleman does that right?

Reply:
I think the format of api documenter changed, but there weren't any api changes, so api extractor didn't force people to regenerate docs